internal/report/report.go

```diff
@@ -261,6 +261,21 @@ func (rpt *Report) newGraph(nodes graph.NodeSet) *graph.Graph {
 		}
 		s.NumLabel = numLabels
 		s.NumUnit = numUnits
+
+		// Also remove pprof::diff=true string tag.
```

Comment:
It would be good to clarify why. Also this code is not super clear. Perhaps make a separate function, and look into how to simplify it. I would also look into whether code could be simpler if the pprof::diff tag was removed in the aggregate() code.

Reply:
Would removing the tag in aggregation() prevent it from being used in later to separate out source and base profiles?

For now, I'm separating this into a different function.

Reply:
Using the SetTag function to clear pprof::diff tag.
And adding comment to explain why this tag shouldn't be set.